packages/iTwinUI-react/src/core/Dialog/Dialog.tsx

```diff
@@ -0,0 +1,56 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import React from 'react';
+import cx from 'classnames';
+import { CommonProps, useTheme } from '../utils';
+import '@itwin/itwinui-css/css/dialog.css';
+import { DialogBackdrop } from './DialogBackdrop';
+import { DialogTitleBar } from './DialogTitleBar';
+import { DialogContent } from './DialogContent';
+
+export type DialogProps = {
+  /**
+   * Content of the dialog.
+   */
+  children: React.ReactNode;
+} & Omit<CommonProps, 'title'>;
```

Comment:
I think we should expose all native element props in every component, i.e. `& React.ComponentProps<'div'>` instead of `& Omit<CommonProps, 'title'>`

Reply:
Exposing